src/main/scripts/jbang.cmd

```diff
@@ -66,7 +66,7 @@ set JBANG_RUNTIME_SHELL=cmd
 2>nul >nul timeout /t 0 && (set JBANG_STDIN_NOTTY=false) || (set JBANG_STDIN_NOTTY=true)
 set "CMD=!JAVA_EXEC!"
 SETLOCAL DISABLEDELAYEDEXPANSION
-%CMD% > "%tmpfile%" %JBANG_JAVA_OPTIONS% -classpath "%jarPath%" dev.jbang.Main %*
+"%CMD%" > "%tmpfile%" %JBANG_JAVA_OPTIONS% -classpath "%jarPath%" dev.jbang.Main %*
```

Comment:
Hi @quintesse, 

It looks like this breaks those who install `jbang` using `chocolatey`. Or I guess anyone using it on Windows with a `JAVA_HOME` path with spaces, which is the most common case if the JDK has been installed with `chocolatey` as well.

Here's some debugging of the issue 
![image](https://user-images.githubusercontent.com/1432287/181656881-b81771d1-e61d-453f-86a1-cfeca30113b0.png)

And here is it back working once I remove the quotes
![image](https://user-images.githubusercontent.com/1432287/181657009-9dcbdab0-3ca1-4a3d-8f5b-d34ffeb2eca6.png)

This is my modified version [jbang.cmd.txt](https://github.com/jbangdev/jbang/files/9214969/jbang.cmd.txt)

Reply:
Hmm, so perhaps that fix should only have been made for the *nix shell script.